components/support/nimbus-fml/src/backends/kotlin/gen_structs/bundled.rs

```diff
@@ -178,6 +187,15 @@ impl CodeType for ImageCodeType {
         }
     }
 
+    fn is_resource_id(&self, _literal: &Literal) -> bool {
+        match _literal {
+            serde_json::Value::String(v) => {
+                is_resource_id(v)
+            }
+            _ => unreachable!("Expecting a string matching an image/drawable resource"),
```

Comment:
This error could  be a little more specific:

```suggestion
            _ => unreachable!("Expecting a string matching an image resource, with pattern [a-z][a-z0-9_]*"),
```